Cesium.externs.js

```diff
@@ -1483,6 +1483,14 @@ Cesium.PolygonOutlineGeometry = function(object) {};
 
 
 /**
+ * @constructor
+ * @param {Object=} object
```

Comment:
A typedef should be created for this parameter based on the options in https://cesiumjs.org/Cesium/Build/Documentation/RectangleOutlineGeometry.html?classFilter=outline#RectangleOutlineGeometry.